packages/front-end/pages/_app.tsx

```diff
@@ -71,34 +75,30 @@ function App({
   useEffect(() => {
     if (!ready) return;
     if (isTelemetryEnabled()) {
-      let _rtQueue: { key: string; on: boolean }[] = [];
+      let _rtQueue: { eventName: string; properties: TrackEventProps }[] = [];
       let _rtTimer = 0;
       gbContext.onFeatureUsage = (key, res) => {
+        const props = {
+          feature: key,
+          revision: "",
+          env: "production",
+          source: res.source,
+          value: res.value,
+          ruleId: res.source === "defaultValue" ? "$default" : res.ruleId || "",
+          variationId: res.experimentResult?.key || "",
+        };
+
         _rtQueue.push({
-          key,
-          on: res.on,
+          eventName: "feature_usage",
```

Comment:
We use `Experiment Viewed` for the other event.  I think we should follow the same pattern and do `Feature Evaluated` here as the event name.